lang/gcc10-bootstrap/Portfile

```diff
@@ -261,21 +258,17 @@ you must use per target compiler. The easy way is using muniversal PG:
 
 PortGroup               muniversal 1.0
 
-if {![info exists universal_possible]} {
```

Comment:
why?

Reply:
Same as above this compact code is already part of Macports-base